src/components/shared/formSignIn/types.ts

```diff
@@ -0,0 +1,12 @@
+export enum InputValidation {
+  Length = 'Name must have at least 2 letters',
+  Invalid = 'That name already exists',
+}
+
+export enum Fields {
+  Name = 'name',
+}
```

Comment:
it is unnecessary to create variables for this

Reply:
This enum is needed for api [queries ](https://prnt.sc/xbYRdM0izdYJ) and I believe it is right to put such parameters into enum, but I agree that we should move this enum to api/types. Am I right?